### ISSUE TITLE
Datepicker: error when selecting month

### ISSUE THREAD

**Issue by MarekLacoAXA** [axa-ch/patterns-library#1181]
Version: 2.2.0

When selecting June (default is July), then an error appears on console.

```
<Datepicker inputfield/>
```

The `day` AND `year` property needs to be set in order to avoid this error:
(unsure how year/day prop relates to defaultValue/date props)
```
<Datepicker inputfield
              year={new Date().getFullYear()}
              day={new Date().getDate()}
/>
```

![image](https://user-images.githubusercontent.com/43809369/61516501-6e15ed80-aa05-11e9-9aa4-c374af1b6836.png)

**Comment by MarekLacoAXA** [comment]
Description updated: workaround found.

**Comment by markus-walther** [comment]
Fixed by NPM release 2.3.0.0.
